.github/workflows/fmt-check.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         os: ["ubuntu-latest"]
     runs-on: ${{ matrix.os }}
     container:
-      image: paritytech/ci-unified:bullseye-1.74.0-2023-11-01-v20231204
```

Comment:
Can you search for `2023-11-01` in the code base and replace them please?  
I know that we have the version hard-coded in some spots.